packages/core/src/lib/relay/index.ts

```diff
@@ -120,30 +116,20 @@ class Relay extends GossipSub implements IRelay {
 
     pushOrInitMapSet(this.observers, contentTopic, observer);
 
-    this.dispatchEvent(
-      new CustomEvent<BasicEventPayload>("observer:added", {
-        detail: {
-          contentTopic,
-        },
-      })
-    );
-
     return () => {
       const observers = this.observers.get(contentTopic);
       if (observers) {
         observers.delete(observer);
-
-        this.dispatchEvent(
-          new CustomEvent<BasicEventPayload>("observer:removed", {
-            detail: {
-              contentTopic,
-            },
-          })
-        );
       }
     };
   }
 
+  public getActiveSubscriptions(): ActiveSubscriptions | undefined {
```

Comment:
```suggestion
  public getActiveSubscriptions(): ActiveSubscriptions {
```
Looks like for now we always return a map. We can evolve the return type later, when needed.

Reply:
Changed on the class level but not changing it on the interface since it will be inherited later from IReceiver.